test/index.js

```diff
@@ -190,3 +192,22 @@ test("works with no styles at all", function(t) {
 
   t.end()
 })
+
+test("works with caching", function(t) {
+  var opts = {path: importsDir, cacheDir: cacheDir}
+  var postcssOpts = {from: "test/fixtures/relative-to-source.css"}
+  var name = "relative-to-source"
+  var actual = postcss()
+    .use(atImport(opts))
+    .process(read("fixtures/" + name), postcssOpts)
+    .css.trim()
+  var expected = read("fixtures/" + name + ".expected")
+  t.equal(actual, expected, "put content in cache")
+  actual = postcss()
+    .use(atImport(opts))
+    .process(read("fixtures/" + name), postcssOpts)
+    .css.trim()
+  expected = read("fixtures/" + name + ".expected")
+  t.equal(actual, expected, "read content from cache")
+  t.end()
+})
```

Comment:
The first `process()` call puts the compiled output [into the cache](https://github.com/andyjansson/postcss-import/blob/master/index.js#L128-140) since `cacheDir` has been specified.

The second `process()` call rather than compiling the CSS again instead [reads it from the cache](https://github.com/andyjansson/postcss-import/blob/master/index.js#L103-111).